src/utils/generator.js

```diff
@@ -61,7 +61,7 @@ export type GeneratorBuildNodeFunction = (Array<BabelNodeExpression>, Serializat
 export type GeneratorEntry = {
   declared?: AbstractValue,
   args: Array<Value>,
-  buildNode: GeneratorBuildNodeFunction,
+  buildNode?: GeneratorBuildNodeFunction,
```

Comment:
Please add a comment here to explain the significance of making the buildNode undefined.